packages/fast-components-react-base/src/tabs/tabs.tsx

```diff
@@ -318,10 +318,13 @@ class Tabs extends Foundation<TabsHandledProps, TabsUnhandledProps, TabsState> {
         switch (e.keyCode) {
             case keyCodeArrowLeft:
             case keyCodeArrowUp:
+                e.preventDefault();
                 this.activateTab(TabLocation.previous);
                 break;
             case keyCodeArrowRight:
             case keyCodeArrowDown:
+                e.preventDefault();
+
                 this.activateTab(TabLocation.next);
                 break;
             case keyCodeHome:
```

Comment:
Didn't even know about the home/end buttons... that's neat. In windows you get scroll up down due to these as well, should you be doing the same for those as well?

Reply:
interesting - is it windows specific?

Reply:
Although, I'm not sure if taking over those two buttons even while within the tabs control is necessarily a good thing.

Reply:
> interesting - is it windows specific?

Don't know -- that's all I have access to now. By scroll up/down, I mean "home"=scroll to the top of the document, and "end"=scroll to the end of the document.

Since these are standard buttons we (I) use in windows to reach the top/end, maybe not take over these, as it could become annoying.

Reply:
Yep, that's my bent here too; thx @prabashn!